docs/core-study-details.md

```diff
@@ -59,6 +59,38 @@ You can see which encounters were ignored as incomplete by examining the
 Usually, you can resolve this by running the ETL process again for the encounters,
 making sure to include all associated resources.
 
+## Optional fields
```

Comment:
Oof, this is reasonable documentation to include, but so easy to get out of date. And it's not like there's a single spot in the source where you could add a comment like "if you add a field, add it to the docs too"...

Though... maybe you _could_ add a note like that to the five or so resource tables we support now?

But regardless, we can be vigilant.

Reply:
i'll add a checkbox item to the PR template - that remains surprisingly effective.